content/5_Plat/Binary_Jump.mdx

```diff
@@ -129,6 +129,37 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```py
+NMAX = 200001
+LOGMAX = 20
```

Comment:
calculate this dynamically with math.log & math.cel

Reply:
Done.